DSESwift/QueryOptions.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  queryOptions.swift
+//  SensePlatform
+//
+//  Created by Tatsuya Kaneko on 01/10/15.
+//
+//
+
+import Foundation
+
+public struct QueryOptions{
+    
+    public enum Interval {
+        case Minute
+        case Hour
+        case Day
+        case Week
+        
+        var description : String {
+            switch self {
+                // Use Internationalization, as appropriate.
+                case .Minute: return "Minute";
+                case .Hour: return "Hour";
+                case .Day: return "Day";
+                case .Week: return "Week";
```

Comment:
@kanekin in the sensor-api spec, this is defined in lowercase. would be nice if we are consistent here.